lib/rollbax.ex

```diff
@@ -127,11 +130,14 @@ defmodule Rollbax do
     Application.get_env(:rollbax, key, default)
   end
 
-  defp fetch_config(key) do
-    case get_config(key, :not_found) do
-      :not_found ->
+  defp fetch_and_resolve_config(key) do
```

Comment:
I renamed the function to make it obvious it does more than just fetch the value.